otsd

```diff
@@ -104,8 +101,12 @@ calendar_path = os.path.expanduser(args.calendar_path)
 calendar = otsserver.calendar.Calendar(calendar_path)
 aggregator = otsserver.calendar.Aggregator(calendar)
 
+proxy = bitcoin.rpc.Proxy()
+info = proxy.getinfo()
+relayfee = float(info['relayfee'])
+
 stamper = otsserver.stamper.Stamper(calendar,
-                                    args.btc_min_relay_feerate / 1000 * bitcoin.core.COIN,
+                                    relayfee / 1000 * bitcoin.core.COIN,
```

Comment:
I saw the loop which retry with at an higher fee if the relay fee is not met, thus is better to remove the option as you say and at this point let's initialize it with the node value.